Upgrade-archive/devenv-methodtype-property-test-codeunits.md

```diff
@@ -23,18 +23,18 @@ Sets the type of method in a test codeunit.
   
 |Value|Syntax example|Purpose|  
 |-----------|-----------------|--------|    
-|**MessageHandler**|<br>`[MessageHandler]` </br> `procedure MessageHandler(Msg : Text[1024]);`|This handler is called when a message function is invoked in the code. The parameter type, **Text**,  contains the text of the function.|  
```

Comment:
it seems to me that here was to much deleted

Reply:
Thanks!